process_email.py

```diff
@@ -926,10 +935,8 @@ def _handle_mail_object(self, mail, email_id, rfc822_email, tmp_dir, start_time_
                 self._parse_email_headers(self._parsed_mail, part, add_email_id=add_email_id)
 
                 # parsed_mail[PROC_EMAIL_JSON_EMAIL_HEADERS].append(part.items())
-
-                # _debug_print("part: {0}".format(part.__dict__))
```

Comment:
Jus remove this if it's not needed anymore

Reply:
I already deleted this line i'm not sure why it's showing up in the diff viewer. If you look at the office365 code tab and switch to my branch you can see it's not there. Must be a bug in githubs viewer